wrangler-core/src/test/java/io/cdap/directives/column/SetTypeTest.java

```diff
@@ -378,13 +415,13 @@ public void testGetOutputSchemaForTypeChangedColumn() throws Exception {
     );
     Schema inputSchema = Schema.recordOf(
       "inputSchema",
-      Schema.Field.of("A", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("B", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("C", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("D", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("E", Schema.of(Schema.Type.INT)),
-      Schema.Field.of("F", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("G", Schema.of(Schema.Type.LONG))
+      Schema.Field.of("A", Schema.unionOf(Schema.of(Schema.Type.STRING), Schema.of(Type.NULL))),
```

Comment:
Can use Schema.nullableOf(Schema.of(Schema.Type.STRING))

Reply:
Done